brightcove_media.module

```diff
@@ -599,37 +619,41 @@ function brightcove_media_list() {
 
   if (!$result) {
     try {
-      $bc = brightcove_initialize();
-      $result = $bc->find($method, $bc_find_params);
+      $client = brightcove_client_load_or_default();
+      list($cms,,) = _brightcove_create_classes($client);
+      $result = $cms->listVideos($search, $sort, $limit, $offset);
```

Comment:
Unless I'm reading this wrong, it looks like $search would be undefined unless `$params['filter']['keywords']` is set. And Where are $sort and $offset getting defined?